ldb/locale/nl/LC_MESSAGES/django.po

```diff
@@ -951,6 +954,11 @@ msgstr "bedrijvenmailing"
 msgid "LDAP username"
 msgstr "LDAP-gebruikersnaam"
 
+#, fuzzy
+#| msgid "LDAP username"
```

Comment:
Je wilt deze fuzzy weghalen. Als het gemarkeerd is als fuzzy wordt het niet meegenomen in de daadwerkelijke vertalingen.